src/System/Hapistrano.hs

```diff
@@ -271,6 +269,9 @@ playScriptLocally cmds =
         }) $
   forM_ cmds $ flip execWithInheritStdout Nothing
 
+-- | Create a file with an initial config file by getting information from the
+-- user.
+
```

Comment:
I'm concerned that this linebreak here might mess with Haddock properly displaying the documentation for this function, is the documentation showing up properly in the generated HTML?

Reply:
That's a good catch. Let's remove the empty line to keep consistency with the rest of the comments. Thanks, David.